heft-plugins/heft-jest-plugin/includes/jest-shared.config.json

```diff
@@ -30,18 +30,21 @@
   "coverageProvider": "v8",
 
   "collectCoverageFrom": [
-    "lib/**/*.js",
+    "lib/**/*.{cjs,js}",
     "!lib/**/*.d.ts",
-    "!lib/**/*.test.js",
+    "!lib/**/*.test.{cjs,js}",
     "!lib/**/test/**",
     "!lib/**/__tests__/**",
     "!lib/**/__fixtures__/**",
     "!lib/**/__mocks__/**"
   ],
   "coveragePathIgnorePatterns": ["/node_modules/"],
 
+  "testEnvironment": "jest-environment-node",
+
   "testEnvironmentOptions": {
-    "url": "http://localhost/"
+    "url": "http://localhost/",
+    "customExportConditions": ["require", "node"]
```

Comment:
![image](https://github.com/microsoft/rushstack/assets/4673363/3d6b9cbf-8527-476f-a02c-119d3e76e1bf)

Should we include the `default` setting?

Reply:
Prefer to let that fail to resolve, instead of resolving and getting a bad version. The two most common conditions in observed packages are "import" and "require".